src/components/createField.jsx

```diff
@@ -373,6 +366,18 @@ export default function connectField(options) {
 
       render() {
         const { props, contextProps } = this
+        
+        /* Render null and log warning in case of formless field */
+        if (!contextProps) {
+          warning(
+            false,
+            'Failed to render the field `%s`: expected to be a child ' +
+              'of a Form component. Please render fields as children of ' +
+              'Form, since formless fields are not currently supported.',
+            props.name,
```

Comment:
Sorry for not addressing this properly the first time, but we can use `this.__fieldPath.join('.')` instead of `props.name` for more descriptive field name. This will help to navigate which field is problematic in case of complex field path.

If there are two fields named `firstName`, but one is in the `groupA` group, and the second is in `groupB` group, it won't be enough to warn using the name only. Joining the field path will yield exact field path like `["groupA", "firstName"]` which would increase developer experience.

Could you please use it instead the name here?

Reply:
Thanks for the quick review, PR updated.